event/workflow.go

```diff
@@ -1,9 +1,87 @@
 package event
 
+import (
+	"github.com/qri-io/qri/profile"
+)
+
 const (
 	// ETWorkflowTrigger signals that a workflow has been triggered
-	// Payload will be a string representation of the `workflow.ID`
-	// of the workflow that should be run
+	// Payload will be a WorkflowTriggerEvent
 	// This event should not block
 	ETWorkflowTrigger = Type("workflow:trigger")
```

Comment:
```go
ETAutomationWorkflowTrigger = Type("automation:WorkflowTrigger")
```

Payload names should stay as is. Generalize this to all event names in this file & rename file to `automation.go`